contracts/src/emergency/sequencer_uptime_feed.cairo

```diff
@@ -0,0 +1,358 @@
+        Aggregator::description()
+    }
+
+    #[view]
+    fn decimals() -> u8 {
+        Aggregator::decimals()
+    }
+
+    #[view]
+    fn type_and_version() -> felt252 {
+        Aggregator::type_and_version()
+    }
+
+    ///
+    /// Ownership
+    ///
+
+    #[view]
+    fn owner() -> ContractAddress {
+        Ownable::owner()
+    }
+
+    #[view]
+    fn proposed_owner() -> ContractAddress {
+        Ownable::proposed_owner()
+    }
+
+    #[external]
+    fn transfer_owernship(new_owner: ContractAddress) {
+        Ownable::transfer_ownership(new_owner)
+    }
+
+    #[external]
+    fn accept_ownership() {
+        Ownable::accept_ownership()
+    }
+
+    #[external]
+    fn renounce_ownership() {
+        Ownable::renounce_ownership()
+    }
+
+    ///
+    /// SimpleReadAccessController
+    ///
+
+    #[external]
+    fn has_access(user: ContractAddress, data: Array<felt252>) -> bool {
+        SimpleReadAccessController::has_access(user, data)
+    }
+
+     #[external]
+    fn check_access(user: ContractAddress) {
+        SimpleReadAccessController::check_access(user)
+    }
+
+    ///
+    /// SimpleWriteAccessController
+    ///
+
+    #[external]
+    fn add_access(user: ContractAddress) {
+        SimpleWriteAccessController::add_access(user)
+    }
+
+    #[external]
+    fn remove_access(user: ContractAddress) {
+        SimpleWriteAccessController::remove_access(user)
+    }
+
+     #[external]
+    fn enable_access_check() {
+        SimpleWriteAccessController::enable_access_check()
+    }
+
+    #[external]
+    fn disable_access_check() {
+        SimpleWriteAccessController::disable_access_check()
+    }
+
+
+    ///
+    /// Internals
+    ///
+
+    fn require_access() {
+        let sender = starknet::info::get_caller_address();
+         SimpleReadAccessController::check_access(sender);
+    }
+
+    fn initializer(initial_status: bool, owner_address: ContractAddress) {
+        SimpleReadAccessController::initializer(owner_address);
+        let round_id = 1_u128;
+        let timestamp = starknet::info::get_block_timestamp();
+        _record_round(round_id, initial_status, timestamp);
+    }
+
+    fn _record_round(round_id: u128, status: bool, timestamp: u64) {
+        _latest_round_id::write(round_id);
+        let block_info = starknet::info::get_block_info().unbox();
+        let block_number = block_info.block_number;
+        let block_timestamp = block_info.block_timestamp;
+
+        let round = RoundTransmission {
+            answer: status,
+            block_num: block_number,
+            started_at: timestamp,
+            updated_at: block_timestamp,
+        };
+        _round_transmissions::write(round_id, round);
+
+        let sender = starknet::info::get_caller_address();
+
+        NewRound(round_id, sender, timestamp);
+        AnswerUpdated(status, round_id, timestamp);
+    }
+
+    fn _update_round(round_id: u128, mut round: RoundTransmission) {
```

Comment:
update round only updates the `updated_at` (aka the transmission timestamp). Remember that the round doesn't increase unless the status of the feed changes -- hence why you don't need to change any other values.